style.css

```diff
@@ -0,0 +1,783 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,400;1,700;1,900&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+  text-decoration: none;
+  list-style: none;
+  font-family: "Lato", sans-serif;
+  box-sizing: border-box;
+}
+
+header,
+.seeHeader,
+.underline {
+  display: none;
+}
+
+.intro,
+.about {
+  background-image: url("./Images/mobile.png");
+  height: auto;
+  text-align: left;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  padding: 0 20px;
+  margin-top: 0;
+}
+
+.actitity p {
+  width: 30%;
+}
+
+.speakerList img {
+  width: 150px;
+  margin-right: 15px;
+}
+
+.Logo p {
+  margin-bottom: 15px;
+}
+
+.past-events p {
+  width: 90%;
+}
+
+.introContainer p:nth-child(1) {
+  padding-top: 20px;
+  font-size: 30px;
+  color: #ec5242;
+}
+
+.activity {
+  display: grid;
+  grid-template-columns: 30% 30% 40%;
+  justify-content: center;
+  align-items: center;
+  padding: 5px;
+  margin: 10px 0;
+  background-color: #36454f;
+}
```

Comment:
- [ ] Let's make sure we stick to the design as much as possible, currently our main section cards have different background color and paragraph's text color is also different from what it is in the design please see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/217835872-797a22eb-68e2-40c0-bbcd-5688b0f3d9f8.png)

how it should be :point_down: 
![image](https://user-images.githubusercontent.com/97521557/217836018-6302c804-9196-40f0-956b-07b495f16a21.png)

same thing with the button :point_down: 
![image](https://user-images.githubusercontent.com/97521557/217836324-ef0c4cd2-24a1-4f61-9e65-5abd54f7e077.png)

please fix this too :+1:

Reply:
Okay.
Thank you.
I have fixed it.